applications/dashboard/controllers/Api/UsersApiController.php

```diff
@@ -0,0 +1,358 @@
+<?php
+/**
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license GPLv2
+ */
+
+use Garden\Schema\Schema;
+use Garden\Web\Data;
+use Garden\Web\Exception\NotFoundException;
+use Garden\Web\Exception\ServerException;
+use Vanilla\Utility\CapitalCaseScheme;
+
+/**
+ * API Controller for the `/users` resource.
+ */
+class UsersApiController extends AbstractApiController {
+
+    /** @var CapitalCaseScheme */
+    private $caseScheme;
+
+    /** @var Schema */
+    private $idParamSchema;
+
+    /** @var UserModel */
+    private $userModel;
+
+    /** @var Schema */
+    private $userPostSchema;
+
+    /** @var Schema */
+    private $userSchema;
+
+    /**
+     * UsersApiController constructor.
+     *
+     * @param UserModel $userModel
+     */
+    public function __construct(UserModel $userModel) {
+        $this->userModel = $userModel;
+        $this->caseScheme = new CapitalCaseScheme();
+    }
+
+    /**
+     * Delete a user.
+     *
+     * @param int $id The ID of the user.
+     * @throws NotFoundException if the user could not be found.
+     * @return null
+     */
+    public function delete($id) {
+        $this->permission('Garden.Users.Delete');
+
+        $in = $this->idParamSchema()->setDescription('Delete a user.');
+        $out = $this->schema([], 'out');
+
+        $this->userByID($id);
+        $this->userModel->deleteID($id);
+
+        return null;
+    }
+
+    /**
+     * Get a schema instance comprised of all available user fields.
+     *
+     * @return Schema Returns a schema object.
+     */
+    protected function fullSchema() {
+        $schema = Schema::parse([
+            'userID:i' => 'ID of the user.',
+            'name:s' => 'Name of the user.',
+            'password:s' => 'Password of the user.',
+            'hashMethod:s' => 'Hash method for the password.',
+            'email:s' => 'Email address of the user.',
+            'photo:s' => [
```

Comment:
This field is not properly transformed before being outputted.

<img width="644" alt="screen shot 2017-08-07 at 3 27 45 pm" src="https://user-images.githubusercontent.com/2412909/29042483-0db95572-7b85-11e7-9d72-053bffcd0e25.png">

It should be a full URL.